core/invoker/src/main/scala/whisk/core/containerpool/ContainerProxy.scala

```diff
@@ -383,7 +387,7 @@ object ContainerProxy {
      * @return a unique container name
      */
     def containerName(prefix: String, suffix: String) =
-        s"wsk_${containerCount.next()}_${prefix}_${suffix}".replaceAll("[^a-zA-Z0-9_]", "")
+        s"wsk${instance.toString}_${containerCount.next()}_${prefix}_${suffix}".replaceAll("[^a-zA-Z0-9_]", "")
```

Comment:
should be `s"wsk${instance.toInt}...` the `toString` is not needed since the value is used in a  String interpolation anyway. This also goes for the change in `InvokerReactive.scala`